lib/plenario2/core/actions/virtual_point_file_actions.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Plenario2.Core.Actions.VirtualPointFieldActions do
+  import Ecto.Query
+  alias Plenario2.Core.Changesets.VirtualPointFieldChangesets
+  alias Plenario2.Core.Schemas.VirtualPointField
+  alias Plenario2.Repo
+
+  def create_from_long_lat(meta_id, longitude, latitude) do
+    params = %{
+      meta_id: meta_id,
+      longitude_field: longitude,
+      latitude_field: latitude
+    }
+
+    VirtualPointFieldChangesets.create_long_lat(%VirtualPointField{}, params)
+    |> Repo.insert()
+  end
+
+  def create_from_loc(meta_id, location) do
+    params = %{meta_id: meta_id, location_field: location}
+
+    VirtualPointFieldChangesets.create_from_loc(%VirtualPointField{}, params)
+    |> Repo.insert()
+  end
+
+  def list_for_meta(meta), do: Repo.all(from f in VirtualPointField, where: f.meta_id == ^meta.id)
```

Comment:
Given that all these fields have a `list_for_meta`, maybe we could have a `FieldAction` or `Field` protocol to formalize this set of common behaviors?

Reply:
Maybe a later PR -- this is a good idea. Let's make an issue for it.

Reply:
Related issue: https://github.com/UrbanCCD-UChicago/plenario2/issues/7